platform/mellanox/mlnx-onie-fw-update.sh

```diff
@@ -67,6 +67,13 @@ show_pending()
 	return $num
 }
 
+system_reboot()
+{
+	echo "Reboot will be done after 5 sec."
```

Comment:
@liat-grozovik this was the original logic of this script, i just recovered it because we need it to fit the new flows. Maybe the reason for that was to allow user to cancel the FW update in the last minute (with ctrl+c) but i am not sure. Also this sleep has nothing to do with platform type because it only postpones the HW reboot: the real work starts only in ONIE mode. Looks like we can completely remove this sleep - no issues with that.